docs/work-with-reports/ViewLaunches.mdx

```diff
@@ -173,6 +173,23 @@ Also, you can use the logs sorting by time, and filtering logs to find a certain
 
 <MediaViewer src="https://youtu.be/_g0GW16sXGU" alt="Log Messages in our automated testing tool" type="video" />
 
+### Timestamps
+
+In ReportPortal version 24.2, microseconds have been added to timestamps to provide a more precise log sequence.
+Previously, some logs shared identical timestamps at the millisecond level, which led to issues with log ordering.
+
+With this update, you have the flexibility to view timestamps in two formats:
+
+1. milliseconds
+
+<MediaViewer src={require('./img/view-launches/Milliseconds.png')} alt="Timestamp in milliseconds" />
+
+2. milliseconds with microseconds
+
+<MediaViewer src={require('./img/view-launches/MillisecondsMicroseconds.png')} alt="Timestamp in milliseconds and microseconds" />
```

Comment:
It will be better to mention that it is a `time with number of microseconds`
From the screenshot:
`414000` - number of microseconds (not `414` or `414,000` that reflects the milliseconds value).
I would also suggest to use another screenshot with non-zeroes microseconds at the end for better visibility for users.